arches_for_science/media/js/views/components/workflows/sample-taking-workflow/sample-taking-sample-location-step.js

```diff
@@ -343,7 +343,7 @@ define([
                 }
             }
 
-            const tilesBelongingToManifest = this.tilesBelongingToManifest(self.card.tiles(), self.canvases());
+            const tilesBelongingToManifest = this.tilesBelongingToManifest(self.card.tiles());
```

Comment:
Unrelated cleanup re #1304; this function no longer takes two args, missed it before merge.